experimental/packages/otlp-transformer/src/metrics/internal.ts

```diff
@@ -96,7 +97,10 @@ export function toMetric(metricData: MetricData): IMetric {
 }
 
 function toSingularDataPoint(
-  dataPoint: DataPoint<number> | DataPoint<Histogram>,
+  dataPoint:
+    | DataPoint<number>
+    | DataPoint<Histogram>
+    | DataPoint<ExponentialHistogram>,
```

Comment:
I think we may be able to reduce this to 
```suggestion
  dataPoint: DataPoint<number>,
```

if we change 

- `toSingularDataPoints(metricData: MetricData)` -> `toSingularDataPoints(metricData: SumMetricData | GaugeMetricData)`
- `toHistogramDataPoints(metricData: MetricData)` -> `toSingularDataPoints(metricData: HistogramMetricData)`

This way, we don't have to export `ExponentialHistogram` from `@opentelemetry/sdk-metrics` and we can merge this PR before following through with #3654, as it would not expose any new types. :slightly_smiling_face:

Reply:
Sorry I was out last week. I wanted to check in to see if this is still needed since #3654 was merged and released?

Reply:
No worries :slightly_smiling_face: 
It's not blocking for me - since there's no release imminent I think it would be safe to merge as-is.